src/main/convert.js

```diff
@@ -123,9 +123,10 @@ export class ThriftFileConverter {
       def.valueType
     )};`;
 
-  generateEnumUnion = (def: Enum) => {
-    return def.definitions.map((d, index) => `"${d.id.name}"`).join(' | ');
-  };
+  generateEnumUnion = (def: Enum) =>
+    !def.definitions.length
```

Comment:
[nit] either invert in here and below or not, be consistent.

Reply:
done